.github/ISSUE_TEMPLATE/ 🐛-bug-report.md

```diff
@@ -0,0 +1,42 @@
+---
+name: "🐛 Bug Report"
+about: Report a bug to fix errors or improve UX
+title: "🐛 Bug Report"
+labels: bug
+assignees: ''
+---
+
+## 🐛 Bug
+
+Describe the issue you encountered.
+
+## 🦋 Expected Behaviour
+
+What should be happening?
```

Comment:
What about the standard "Expected Behaviour" vs "Observed Behaviour" contrast? I've always seen that, and it made sense to me as very clear :)